playbooks/roles/mysql/tasks/main.yml

```diff
@@ -1,17 +1,29 @@
+---
+
 # edX is currently using MySQL 5.6, which is not available on Xenial by default.
 - name: Add PPA for MySQL 5.6
   apt_repository:
     repo: "ppa:ondrej/mysql-5.6"
     update_cache: yes
 
```

Comment:
The MySQL role currently does not run, since it does not install the correct lib for Python 3.  This is a bug in the MySQL role we are using, and it is fixed in the latest version.  However, upgrading to the latest version requires more thorough testing, so I included this task here:
```
# This task can be removed once we move to the latest version of the geerlingguy.mysql role.  The
# version we are currently using only installs the Python2 version of python-mysqldb.
- name: Install python3-mysqldb package
  apt:
    name: python3-mysqldb
```
Note that it is not safe to run the MySQL role against the currently running servers, since setting up the replication is done in multiple steps, and running the role would run the earlier steps again – see Josh's comments on the ticket.  This means this fix is not particularly important, since we can't run the role anyway.  In the long run this needs to be sorted out, but that's out of scope for the current work.

Reply:
Ok, thanks for this info. I'll put that task there, and also be careful to not run the full role, but rather only step through it to install the nginx proxy and the exporter.

Reply:
Actually, to be more specific, I believe our `mysql` role is safe to run, but our fork of the external `geerlingguy.mysql` role isn't.  Please double-check this info before running against the production servers. :)

Reply:
If you noticed that even one of them wasn't safe, that's enough for me :p I won't run the full roles, but rather only use `--step` to commit the tasks that that we want, e.g. the nginx-proxy and the exporter parts. I did that earlier for HAProxy stage when testing so it should be good.